test/e2e/storage/drivers/in_tree.go

```diff
@@ -1314,150 +1313,6 @@ func (v *vSphereVolume) DeleteVolume(ctx context.Context) {
 	v.nodeInfo.VSphere.DeleteVolume(v.volumePath, v.nodeInfo.DataCenterRef)
 }
 
-// Azure Disk
-type azureDiskDriver struct {
-	driverInfo storageframework.DriverInfo
-}
-
-type azureDiskVolume struct {
-	volumeName string
-}
-
-var _ storageframework.TestDriver = &azureDiskDriver{}
-var _ storageframework.PreprovisionedVolumeTestDriver = &azureDiskDriver{}
-var _ storageframework.InlineVolumeTestDriver = &azureDiskDriver{}
-var _ storageframework.PreprovisionedPVTestDriver = &azureDiskDriver{}
-var _ storageframework.DynamicPVTestDriver = &azureDiskDriver{}
-var _ storageframework.CustomTimeoutsTestDriver = &azureDiskDriver{}
-
-// InitAzureDiskDriver returns azureDiskDriver that implements TestDriver interface
-func InitAzureDiskDriver() storageframework.TestDriver {
```

Comment:
Please keep the tests for now, when we still have Azure SDK. When removing the AzureFile + cloud provider (+ azure SDK deps), still keep just the parts that don't require the SDK - the dynamic provisioning with in-tree AzureDisk / AzureFile should work forever and should not require any special deps. Check AWS or Cinder TestDrivers in this file.